.binstar.yml

```diff
@@ -6,7 +6,6 @@ platform:
   - win-64
   - win-32
 engine:
-  - python=2.6
```

Comment:
Given that I'm not actively doing much of the work on Toolz these days I'll
defer to your opinion.

However, I think that if we do drop Python 2.6 then we should first issue a
call out to the general community and have people weigh in.  Perhaps raise
an issue and we tweet about it?

On Mon, Aug 22, 2016 at 10:39 AM, Erik Welch <notifications@github.com>
wrote:

> In .binstar.yml
> <https://github.com/pytoolz/toolz/pull/326#discussion_r75689598>:
>
> > @@ -6,7 +6,6 @@ platform:
> >    - win-64
> >    - win-32
> >  engine:
> > -  - python=2.6
>
> I guess so. It would be straightforward to support 2.6. It was easier to
> drop it. What's your preference?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/pytoolz/toolz/pull/326/files/0ddc67ff372b46ab8c7a406f05809f016008b9a4#r75689598>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASszEM2Dh5P_XTaXGDIzpqZrSV8aQBNks5qibSXgaJpZM4JpYoz>
> .
>